amazon_transcribe/model.py

```diff
@@ -224,6 +225,8 @@ class StartStreamTranscriptionRequest:
         results. A higher stability level means that the transcription results
         are less likely to change. Higher stability levels can come with lower
         overall transcription accuracy.
+    :param language_model_name
```

Comment:
This will also need a closing `:`.

```suggestion
    :param language_model_name:
```